flink-streaming-java/src/main/java/org/apache/flink/streaming/api/transformations/OneInputTransformation.java

```diff
@@ -42,7 +42,7 @@
  * @param <OUT> The type of the elements that result from this {@code OneInputTransformation}
  */
 @Internal
-public class OneInputTransformation<IN, OUT> extends PhysicalTransformation<OUT> {
+public class OneInputTransformation<IN, OUT> extends TransformationWithLineage<OUT> {
```

Comment:
I can see that some PhysicalTransformations now subclass TransformationWithLineage. How did you decide which ones to use? I am thinking we would want lineage for MultipleInputTransformation, KeyedMultipleInputTransformation, TwoInputTransformation and maybe others?

Reply:
MultipleInputTransformation, KeyedMultipleInputTransformation, TwoInputTransformation  are internal transformations. They will be useful for propagate the column level lineage info. But It is not in the scope of this PR.

Reply:
Looks like `LineageGraphUtils` only handles Source/Sink transformations, why making `OneInputTransformation` extend `TransformationWithLineage`?